### DISCUSSION TITLE
Back button

### DISCUSSION THREAD

**Discussion by ZeljkoBG** [osmandapp/OsmAnd#19754] [General]
When I go to the settings

Menu/Settings/bla-bla/bla-bla/Rute ... and I press the Back button, to look at **the previous window**, the application returns me to MENU (start).

**Comment by Sonwon1** [comment]
Yes, this is annoying and only one inconsistency in the user interface. Some screens are missing a back button. And I would add an exit to map button if the back button doesn't take you to the map. Actually the back button should take you back to the menu so we could use an exit to map button.

**Comment by ZeljkoBG** [comment]
The problem has existed for a long time. I see that others do not see that it is a huge problem.
I started a similar topic, here [#19229 ](https://github.com/osmandapp/OsmAnd/discussions/19229#discussion-6304090)
